packages/plugin-selenium-driver/src/plugin/index.ts

```diff
@@ -288,6 +288,15 @@ export class SeleniumPlugin implements IBrowserProxyPlugin {
         }
     }
 
+    public async newWindow(applicant: string, val: string, windowName: string, windowFeatures: string) {
+        await this.createClient(applicant);
+        const client = this.getBrowserClient(applicant);
+
+        if (client) {
+            return this.wrapWithPromise(client.newWindow(val, windowName, windowFeatures));
```

Comment:
In closest feature we will have puppeteer plugin too, so we can't use `windowFeatures`, it's too specific.
You can pass it like an object 
```
{
    width: 1000,
    height: 500,
}
```
and convert to string inside this function.

Reply:
Sorry, I don't catch you means. could you please help refactor this part of code?

Reply:
I think argument `windowFeatures` should be an object, not string.
You can transform it to string inside this function later.

Reply:
done